### PR TITLE
Application: Remove unused dirtyRegions variable

### PR DESCRIPTION
**Pull request by popcornmix**
## Description
Remove unused dirtyRegions variable

## How Has This Been Tested?
In last night's Milhouse build

## Types of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by popcornmix** [comment]
ping @FernetMenta

**Comment by popcornmix** [comment]
jenkins build this please

File: xbmc/Application.cpp [hunk 1]
```diff
@@ -1873,12 +1873,9 @@ void CApplication::Render()
   if(!g_Windowing.BeginRender())
     return;
 
-  CDirtyRegionList dirtyRegions;
-
   // render gui layer
   if (!m_skipGuiRender)
   {
-    dirtyRegions = g_windowManager.GetDirty();
```
Comment (FernetMenta) [inline xbmc/Application.cpp]: did you check GetDirty()? is it const or does it change state? is it used elsewhere or can it be dropped?

Comment (popcornmix) [inline xbmc/Application.cpp]: Actually I think GetDirty can be removed.
GetDirty returns ``m_tracker.GetDirtyRegions()``
Seems the code that uses GetDirtyRegions is now:
https://github.com/xbmc/xbmc/blob/master/xbmc/guilib/GUIWindowManager.cpp#L1094

**Comment by popcornmix** [comment]
jenkins build this please

**Comment by FernetMenta** [comment]
+1

**Pull request merged**